.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+/** @type {import("prettier").Options} */
+export default {
+  semi: true,
+  singleQuote: false,
+  trailingComma: "all",
+  printWidth: 120,
+};
```

Comment:
It seems that other repositories in this organization uses single quote? 🤔

Reply:
Oh, was that the case? I hadn’t checked that. My intention was to align the writing style as closely as possible with Rust. To me, representing strings with single quotes feels a bit odd, as many languages use single quotes for char.

That’s why I chose double quotes, but if it’s not appropriate for the project or organization’s operations, please feel free to make changes!”

Reply:
@Boshen should make a final decesion, though.

Reply:
See #33